environment.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Is'n more general name `bootstrap` will be better? In that case we can remove `require 'autoload.php'` from all other entry points. And if user needs own custom logic it will be likely placed in such file